cli/tools/fmt.rs

```diff
@@ -819,8 +819,10 @@ pub fn format_file(
     "css" | "scss" | "sass" | "less" => {
       format_css(file_path, file_text, fmt_options)
     }
-    "html" => format_html(file_path, file_text, fmt_options, unstable_options),
-    "svelte" | "vue" | "astro" | "vto" | "njk" => {
+    "html" | "xml" | "svg" => {
```

Comment:
@dsherret AFAICT without changing `deno_media_type` and specifying `Xml` and `Svg` types we won't be able to format template strings like:
```

xml`<CATALOG></CATALOG>`

svg`<svg height="280" width="360" xmlns="http://www.w3.org/2000/svg">...`

Reply:
We should remove deno_media_type being used for this. It just clutters it.